common/bindfstools.py

```diff
@@ -0,0 +1,112 @@
+#    Copyright (C) 2016 Taylor Raack
+#
+#    This program is free software; you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation; either version 2 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License along
+#    with this program; if not, write to the Free Software Foundation, Inc.,
+#    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import os
+import grp
+import subprocess
+import gettext
+
+import config
+from mount import MountControl
+import logger
+import tools
+from exceptions import MountException
+
+_=gettext.gettext
+
+# TODO - Move some common methods (_umount() and check_fuse()) into the base class
+class Local(MountControl):
+    """
+    Mount local path with bindfs. This allows for read-only viewing of snapshots stored on a locally mounted filesystem.
+    """
+
+    CHECK_FUSE_GROUP = False
```

Comment:
I moved this into `MountControl` base-class, too. `common/configure` will change it only in `mount.py` based on the givven arguments `--fuse-group` or `--no-fuse-group`. This would break it.